production/helm/loki/templates/_helpers.tpl

```diff
@@ -42,3 +42,13 @@ Create the name of the service account
 {{- end -}}
 {{- end -}}
 
+{{/* Generate basic labels */}}
+{{- define "loki.labels" }}
+app: {{ template "loki.name" . }}
```

Comment:
Should probably just inline this unless you're going to change every resource to use `loki.labels`